jlibra-examples/src/main/java/dev/jlibra/example/AsyncTransferExample.java

```diff
@@ -0,0 +1,143 @@
+        BCEdDSAPrivateKey privateKeySource = (BCEdDSAPrivateKey) keyPairSource.getPrivate();
+        BCEdDSAPublicKey publicKeySource = (BCEdDSAPublicKey) keyPairSource.getPublic();
+        AccountAddress source = AccountAddress.ofPublicKey(publicKeySource);
+        ExampleUtils.mint(source, 20L * 1_000_000L);
+
+        KeyPair keyPairTarget = kpGen.generateKeyPair();
+        BCEdDSAPublicKey publicKeyTarget = (BCEdDSAPublicKey) keyPairTarget.getPublic();
+        AccountAddress target = AccountAddress.ofPublicKey(publicKeyTarget);
+
+        // sleep for 1 sec to make sure the minted money is available in the account.
+        // Sometimes the faucet api is working slowly and you might need to increase the
+        // time.
+        Thread.sleep(1000);
+
+        ManagedChannel channel = ManagedChannelBuilder.forAddress("ac.testnet.libra.org", 8000)
+                .usePlaintext()
+                .build();
+        AdmissionControl admissionControl = new AdmissionControl(channel);
+
+        List<CompletableFuture<SubmitTransactionResult>> transactions = new ArrayList<>();
+
+        logger.info("Start creating transactions..");
+        for (int i = 0; i < 10; i++) {
+            U64Argument amountArgument = new U64Argument(1 * 1000000);
+            AccountAddressArgument addressArgument = new AccountAddressArgument(target.getByteSequence());
+            Transaction transaction = ImmutableTransaction.builder()
+                    .sequenceNumber(i)
+                    .maxGasAmount(140000)
+                    .gasUnitPrice(0)
+                    .senderAccount(AccountAddress.ofPublicKey(publicKeySource))
+                    .expirationTime(Instant.now().getEpochSecond() + 60)
+                    .payload(ImmutableScript.builder()
+                            .code(Move.peerToPeerTransferAsBytes())
+                            .addArguments(addressArgument, amountArgument)
+                            .build())
+                    .build();
+            SignedTransaction signedTransaction = ImmutableSignedTransaction.builder()
+                    .publicKey(publicKeySource)
+                    .transaction(transaction)
+                    .signature(ImmutableSignature.builder()
+                            .privateKey(privateKeySource)
+                            .transaction(transaction)
+                            .build())
+                    .build();
+            transactions.add(admissionControl
+                    .asyncSubmitTransaction(signedTransaction));
+            logger.info("Created transaction {}, sending 1 libra from {} to {}", i, source.getByteSequence(),
+                    target.getByteSequence());
+        }
+
+        logger.info("All transactions created. Wait for them to be accepted..");
+        long time = System.currentTimeMillis();
+        CompletableFuture.allOf(transactions.toArray(new CompletableFuture[transactions.size()])).get();
```

Comment:
You could avoid calling `get()` here. If you skip it, you can remove `Thread.sleep` but instead chain this `CompletableFuture` with next one using `thenCompose` to chain `CompletableFuture<UpdateToLatestLedgerResult> result`